source/engine/ui/editor/EditorUI.cpp

```diff
@@ -0,0 +1,47 @@
+#include "EditorUI.h"
+#include "ncengine/ecs/Registry.h"
+#include "ncengine/ui/ImGuiUtility.h"
+#include "ncengine/window/Window.h"
+
+#define RUN_ONCE(f) { [[maybe_unused]] static auto _ = [](){f; return true;}(); }
```

Comment:
A comment here explaining what this does for those that haven't seen Daisy's talk would be very useful